src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -119,17 +119,13 @@ VPNFlickable {
 
                 onClicked: {
                     VPN.recordGleanEvent("deleteAccountRequested");
+                    navbar.visible = false
```

Comment:
Even though it is a somewhat common pattern in this project, I don't love referencing ancestor components in other files like this since it's very fragile, still considering other implementations

Reply:
You could also do this by adding an objectName to the parent Item {} in `ViewDeleteAccount.qml` and then adding something along the lines of `&& mainStackView.currentItem.objectName !== "viewDeleteAccountObjectName"` to toggle the nav bar visibility in main.qml.

Reply:
haha seems hacky, but probably better than all the rebinding